core/src/main/scala/me/jeffmay/neo4j/client/TxnRef.scala

```diff
@@ -4,7 +4,20 @@ import play.api.libs.json.{JsString, Writes}
 
 import scala.util.{Failure, Success, Try}
 
-case class TxnRef(url: String, domain: String, id: Int)
+/**
+  * A reference to an open transaction.
+  *
+  * @see <a href="http://neo4j.com/docs/stable/rest-api-transactional.html">Transaction Documentation</a>
+  *
+  * @note Open transactions are not shared among members of an HA cluster.
+  *       Therefore, if you use this endpoint in an HA cluster, you must ensure that
+  *       all requests for a given transaction are sent to the same Neo4j instance.
+  *
+  * @param url the URL to commit the transaction
+  * @param host the host address of the Neo4j instance on which the transaction is open
```

Comment:
URL does consist of the host. I just wanted to allow access to the original value for logging and compatibility purposes. Splitting it up is just to enable more intelligent handling of transactions (like grouping by host or creating a map with the integer transaction id as a key). Probably not that useful.